fluss-client/src/main/java/com/alibaba/fluss/client/admin/FlussAdmin.java

```diff
@@ -277,6 +279,21 @@ public CompletableFuture<List<PartitionInfo>> listPartitionInfos(TablePath table
                 .thenApply(ClientRpcMessageUtils::toPartitionInfos);
     }
 
+    @Override
+    public CompletableFuture<List<PartitionInfo>> listPartitionInfos(
+            TablePath tablePath, PartitionSpec partitionSpec) {
+        ListPartitionInfosRequest request = new ListPartitionInfosRequest();
+        PbPartitionSpec pbPartitionSpec = ClientRpcMessageUtils.makePbPartitionSpec(partitionSpec);
```

Comment:
Change to static import: `ClientRpcMessageUtils.makePbPartitionSpec(partitionSpec)` -> `makePbPartitionSpec(partitionSpec)`

In additional, I found that `ClientRpcMessageUtils.makePbPartitionSpec(partitionSpec)` can also be used in `ClientRpcMessageUtils#makeCreatePartitionRequest()` and  `ClientRpcMessageUtils#makeDropPartitionRequest()` to avoid duplicated code. Can you modify it?